sites/fast-website/src/app/css/style.css

```diff
@@ -361,3 +360,9 @@ fast-divider::after {
     display: none;
   }
 }
+
+@media screen and (max-width: 750px) {
+	.community-section {
+		padding-bottom: var(--navbar-height);
+	}
```

Comment:
Is it just me or is the spacing off here a bit?

Reply:
Gah!  It keeps using tabs for some reason.  Will fix right now.

Reply:
Fixed